src/app/parcels/parcelsTable/fetchParcelTableData.ts

```diff
@@ -263,18 +264,20 @@ export const fetchParcelStatuses = async (): Promise<ParcelStatusesReturnType> =
     return { data: parcelStatusesList, error: null };
 };
 
-export const getClientIdForSelectedParcel = async (parcelId: string): Promise<string> => {
-    const { data, error } = await supabase
-        .from("parcels")
+export const getClientIdForSelectedParcel = async (
```

Comment:
I moved this function into this file and I handled the error instead of throwing it